OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -450,5 +453,22 @@ public void SetScaleModifier(float scale)
 			scaleModifier = scale;
 			OnWindowScaleChanged(windowScale, windowScale * oldScaleModifier, windowScale, windowScale * scaleModifier);
 		}
+
+		public IReadOnlyDictionary<string, int> GetDisplays()
```

Comment:
Imo better to have this method returning a list of `Pair<string, int>`.

Reply:
Is there a reason why this cant return a string[]?

Reply:
Its using the number here later in the call stack https://github.com/OpenRA/OpenRA/pull/17648/files#diff-e2bdca10e38c0db34ae3c2d91cd2fc48R136

Reply:
Yeah, but unless im missing something the number is just the array index?

Reply:
true

Reply:
I agree, the change to an array is no problem.